admin_guide/topics/testing-auto-scaler.adoc

```diff
@@ -0,0 +1,154 @@
+    app: scale-up
+spec:
+  replicas: 20 <1>
+  selector:
+    matchLabels:
+      app: scale-up
+  template:
+    metadata:
+      labels:
+        app: scale-up
+    spec:
+      containers:
+      - name: origin-base
+        image: openshift/origin-base
+        resources:
+          requests:
+            memory: 2Gi
+        command:
+        - /bin/sh
+        - "-c"
+        - "echo 'this should be in the logs' && sleep 86400"
+      terminationGracePeriodSeconds: 0
+----
++
+<1> This deployment specifies 20 replicas, but the initial size of the cluster
+cannot run all of the pods without first increasing the number of compute nodes.
+
+. Create a namespace for the deployment:
++
+[source,bash]
+----
+$ oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: autoscaler-demo
+EOF
+----
+
+. Deploy the configuration:
++
+[source,bash]
+----
+$ oc apply -n autoscaler-demo -f scale-up.yaml
+----
++
+. View the pods in your namespace:
+.. View the running pods in your namespace:
++
+[source,bash]
+----
+$ oc get pods -n autoscaler-demo | grep Running
+cluster-autoscaler-5485644d46-ggvn5   1/1       Running   0          1d
+scale-up-79684ff956-45sbg             1/1       Running   0          31s
+scale-up-79684ff956-4kzjv             1/1       Running   0          31s
+scale-up-79684ff956-859d2             1/1       Running   0          31s
+scale-up-79684ff956-h47gv             1/1       Running   0          31s
+scale-up-79684ff956-htjth             1/1       Running   0          31s
+scale-up-79684ff956-m996k             1/1       Running   0          31s
+scale-up-79684ff956-pvvrm             1/1       Running   0          31s
+scale-up-79684ff956-qs9pp             1/1       Running   0          31s
+scale-up-79684ff956-zwdpr             1/1       Running   0          31s
+----
+.. View the pending pods in your namespace:
++
+[source,bash]
+----
+$ oc get pods -n autoscaler-demo | grep Pending
+scale-up-79684ff956-5jdnj             0/1       Pending   0          40s
+scale-up-79684ff956-794d6             0/1       Pending   0          40s
+scale-up-79684ff956-7rlm2             0/1       Pending   0          40s
+scale-up-79684ff956-9m2jc             0/1       Pending   0          40s
+scale-up-79684ff956-9m5fn             0/1       Pending   0          40s
+scale-up-79684ff956-fr62m             0/1       Pending   0          40s
+scale-up-79684ff956-q255w             0/1       Pending   0          40s
+scale-up-79684ff956-qc2cn             0/1       Pending   0          40s
+scale-up-79684ff956-qjn7z             0/1       Pending   0          40s
+scale-up-79684ff956-tdmqt             0/1       Pending   0          40s
+scale-up-79684ff956-xnjhw             0/1       Pending   0          40s
+----
++
+These pending pods have nowhere to run. In the background, the cluster
+auto-scaler automatically provisions new compute nodes to run the pods on. It
+can several minutes for the nodes have a `Ready` state in the cluster.
+
```

Comment:
@kalexand-rh 
_These pending pods have nowhere to run_ sounds a little too cute. 
How about combining the first two sentenced_The pods cannot be scheduled until the auto scaler provisions new nodes_
(Poor pods, nowhere to run....)